tests/titback/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Part of the Tit Solver project, under the MIT License.
+# See /LICENSE.md for license information. SPDX-License-Identifier: MIT
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+add_tit_test(
+  titback
+  NAME "titback/exec"
+  MATCH_STDOUT "exec_stdout.txt"
+  COMMAND titback -c "import numpy $<SEMICOLON> print(numpy)"
```

Comment:
It maybe rather inconvenient to use `$<SEMICOLON>` everywhere. We should think of a better way of using semicolons in function arguments, if there will be multiple occurrences of such situations. For now, let's add a simple TODO.